install_config/aggregate_logging.adoc

```diff
@@ -109,7 +115,7 @@ before choosing parameters:
 By default the Elasticsearch service uses port 9300 for TCP communication
 between nodes in a cluster.
 ====
-
+[NOTE]
```

Comment:
Is this [NOTE] needed?

Reply:
Leaving this, as it has been in the file, and I did not change it.